### PR TITLE
Zachmu/cheat sheet

### PR DESCRIPTION
**Pull request by zachmu**

### CODE REVIEW

File: packages/dolt/content/introduction/cheat-sheet.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+| ---                                                   | -----                             | --------                                                   |
+| `CALL DOLT_REMOTE('add', 'myRemote', 'myOrg/myRepo')` | `dolt remote add myRemote/myRepo` | Adds a new DoltHub remote                                  |
+| `SELECT * FROM dolt_remotes`                          | `dolt remote`                     | Lists remotes                                              |
+| `CALL DOLT_FETCH()`                                   | `dolt fetch`                      | Fetches all branches from the remote                       |
+| `CALL DOLT_PULL()`                                    | `dolt pull`                       | Fetch and merge commits from the remote tracking branch    |
+| `CALL DOLT_PUSH('origin', 'myBranch')`                | `dolt push origin myBranch`       | Push local commits of branch `myBranch` to remote `origin` |
+| `CALL DOLT_PUSH()`                                    | `dolt push`                       | Push local commits to the reote tracking branch            |
```
Comment (timsehn) [inline packages/dolt/content/introduction/cheat-sheet.md]:
```suggestion
| `CALL DOLT_PUSH()`                                    | `dolt push`                       | Push local commits to the remote tracking branch            |
```

**Review by timsehn — APPROVED** [review]
Found just one typo.

File: packages/dolt/content/introduction/cheat-sheet.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+| `CALL DOLT_REBASE('--interactive', 'main')`                              | `dolt rebase --interactive main`                    | Begins an interactive rebase session                                                                             |
+| `SELECT * FROM dolt_reflog(mybranch')`                                   | `dolt reflog mybranch`                              | Shows the history of a ref, included deleted refs                                                                |
+| `SELECT * FROM dolt_commit_ancestors where commit_hash = HASHOF('main')` | No equivalent                                       | Shows the parent commit(s) of a commit                                                                           |
+| `SELECT * MERGE_BASE('main', 'feature')`                                 | No equivalent                                       | Shows the common ancestor of two commits                                                                         |
```
Comment (tbantle22) [inline packages/dolt/content/introduction/cheat-sheet.md]:
```suggestion
| `SELECT * DOLT_MERGE_BASE('main', 'feature')`                                 | No equivalent                                       | Shows the common ancestor of two commits                                                                         |
```

File: packages/dolt/content/introduction/cheat-sheet.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+| `CALL DOLT_VERIFY_CONSTRAINTS()`                                         | `dolt verify-constraints`                           | Checks for constraint violations (e.g. after checks had been disabled)                                           |
+| `CALL DOLT_GC()`                                                         | `dolt gc`                                           | Runs garbage collection to compact the size of the database on disk                                              |
+| `CALL DOLT_REBASE('--interactive', 'main')`                              | `dolt rebase --interactive main`                    | Begins an interactive rebase session                                                                             |
+| `SELECT * FROM dolt_reflog(mybranch')`                                   | `dolt reflog mybranch`                              | Shows the history of a ref, included deleted refs                                                                |
```
Comment (tbantle22) [inline packages/dolt/content/introduction/cheat-sheet.md]:
```suggestion
| `SELECT * FROM dolt_reflog('mybranch')`                                   | `dolt reflog mybranch`                              | Shows the history of a ref, included deleted refs                                                                |
```

File: packages/dolt/content/introduction/cheat-sheet.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+| `CALL DOLT_CHECKOUT('myBranch')`               | `dolt checkout myBranch`    | Switches to another branch                  |
+| `CALL DOLT_CHECKOUT('-b', 'myBranch')`         | `dolt checkout -b myBranch` | Creates a new branch and switches to it     |
+| `CALL DOLT_MERGE('myBranch')`                  | `dolt merge mybranch`       | Merges a branch into the checked out branch |
+
+## Diffing
+
+| SQL server                                                   | Dolt CLI                            | Comments                                                  |
+| ---                                                          | -----                               | --------                                                  |
+| `SELECT * FROM dolt_diff('HEAD', 'WORKING', 'mytable')`      | `dolt diff mytable`                 | Shows the working diff for `mytable`                      |
+| `SELECT * FROM dolt_diff_stat('HEAD', 'WORKING', 'mytable')` | `dolt diff --summary mytable`       | Shows statistics for the diff of `mytable`                |
```
Comment (tbantle22) [inline packages/dolt/content/introduction/cheat-sheet.md]: dolt diff summary shows a summary of data and schema changes. Could be worth mentioning here too?
```suggestion
| `SELECT * FROM dolt_diff_stat('HEAD', 'WORKING', 'mytable')` | `dolt diff --stat mytable`       | Shows statistics for the diff of `mytable`                |
```

File: packages/dolt/content/SUMMARY.md [hunk 5]
```diff
@@ -24,6 +24,7 @@
   - [Audit](introduction/use-cases/audit.md)
   - [Configuration Management](introduction/use-cases/configuration-management.md)
   - [Offline First](introduction/use-cases/offline-first.md)
+- [Cheat Sheet](introduction/cheat-sheet.md)
```
Comment (timsehn) [inline packages/dolt/content/SUMMARY.md]: I think maybe this should be in Guides

**Pull request merged**